ch32fun/ch32fun.mk

```diff
@@ -394,7 +376,7 @@ gdbclient :
 	gdb-multiarch $(TARGET).elf -ex "target remote :3333"
 
 clangd :
-	make clean
+	$(MAKE) clean
```

Comment:
Can you put a MAKE?=make earlier? so on old Windows Make versions it still works?